composer_deploy.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Validate arguments passed to the script
+if [[ $# -eq 1 && -n "$1" && -d "$1" ]]; then
+    script_path=$(cd $(dirname $0) && pwd)
+    project_path=$1
+else
+    echo "Usage: bash composer_deploy.sh <project_path>"
+    exit 1
+fi
+
+# Copy only required files to the project directory
+mkdir -p "$1"/docker
+cp -r "${script_path}"/elasticsearch "$1"/docker/
+cp -r "${script_path}"/nginx "$1"/docker/
+cp -r "${script_path}"/php "$1"/docker/
+cp "${script_path}"/.dockerignore "$1"/docker/
+cp "${script_path}"/.env.dist "$1"/docker/
+cp "${script_path}"/docker-compose.yml "$1"/docker/
+cp "${script_path}"/Makefile "$1"/docker/
+
+# Install the Makefile in the project directory
+if [[ ! -f "$1"/Makefile ]]; then
+    cp "${script_path}"/Makefile.sample "$1"/Makefile
```

Comment:
This Makefile is auto included ? 
https://github.com/EmakinaFR/docker-magento2/pull/19/files#diff-99ab1dfb6c4b62694bd5289570810afdR7

Reply:
Not sure to understand the question. 😄